### PR TITLE
[IOTDB-2836] Fix ErrorHandlingUtils exception stacktrace logging

### PR DESCRIPTION
**Pull request by ericpai**
See JIRA: https://issues.apache.org/jira/browse/IOTDB-2836

### CODE REVIEW

**Comment by ericpai** [comment]
@HTHou PTAL

File: server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java [hunk 1]
```diff
@@ -150,7 +151,7 @@ public static TSStatus tryCatchNonQueryException(Exception e) {
       // ignore logging sg not ready exception
       if (!(rootCause instanceof StorageGroupNotReadyException)) {
         if (((IoTDBException) e).isUserException()) {
-          LOGGER.warn(message + e.getMessage());
+          LOGGER.warn(message, e);
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:154]: Why do you change this? I think there is a reason for printing `e.getMessage()` only... @HeimingZ

Comment (ericpai) [reply server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:154]: I'm not sure whether we don't need the stackstace indeed.

Comment (HeimingZ) [reply server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:154]: @jun0315 PTAL

Comment (jun0315) [reply server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:154]: In the original version:
https://github.com/apache/iotdb/pull/4316/files#diff-94a8b388744fb4d4422cb90e8fd8650803610de8ea6e734b3924f2d3a7cb86f4L2318 (In this page, we need click the `TSServiceImpl.java` `Load diff` button)
it was printed `e.getMessage()`, which I transplanted directly at that time. 
But now I think maybe need the stackstace is a better way?
LGTM

**Review by jun0315 — APPROVED** [review]

File: server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java (hunk 1 above)
Comment (ericpai) [reply server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:154]: Thanks @jun0315

File: server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java [hunk 2]
```diff
@@ -150,7 +151,7 @@ public static TSStatus tryCatchNonQueryException(Exception e) {
...
         } else {
           LOGGER.warn(message, e);
         }
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java:157]: Ok, and then we don't need the if else here?

Comment (ericpai) [reply server/src/main/java/org/apache/iotdb/db/utils/ErrorHandlingUtils.java]: Good catch! I removed the `if-else` statements.

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**